lib/install/validate/validator.go

```diff
@@ -290,15 +290,11 @@ func (v *Validator) checkVDSMembership(ctx context.Context, network string) erro
 		return err
 	}
 
-	nets, err := v.getNetworks(ctx, network)
+	net, err := v.getNetwork(ctx, network)
```

Comment:
alright, I think you don't have to call this method in this membership checking, cause all networks are already checked ahead of your method call